--- Makefile.orig	2013-12-02 08:51:32.000000000 -0800
+++ Makefile	2013-12-02 08:52:12.000000000 -0800
@@ -9,8 +9,6 @@
 INSTALLDIR = /usr/local/bin
 SRCDIR = src
 EXPDIR = exp
-export CUDD_ROOT = extern/cudd-2.5.0
-CUDD_LIB = $(CUDD_ROOT)/cudd/libcudd.a $(CUDD_ROOT)/mtr/libmtr.a $(CUDD_ROOT)/st/libst.a $(CUDD_ROOT)/util/libutil.a $(CUDD_ROOT)/epd/libepd.a
 
 LEX = flex
 LFLAGS = 
@@ -20,9 +18,9 @@
 CC = gcc
 LD = ld -r
 
-export CFLAGS_BASE = -g -Wall -pedantic -ansi -DHAVE_IEEE_754 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8
-CFLAGS = $(CFLAGS_BASE) -I$(CUDD_ROOT)/include -Isrc
-LDFLAGS = $(CUDD_LIB) -lm
+export CFLAGS_BASE = -g -Wall -pedantic -ansi -DHAVE_IEEE_754 -DSIZEOF_VOID_P=4 -DSIZEOF_LONG=4
+CFLAGS = $(CFLAGS_BASE) -I/opt/local/include/cudd -Isrc
+LDFLAGS = -L/opt/local/lib/cudd -lcudd -lm
 
 # To use and statically link with GNU Readline
 #CFLAGS += -DUSE_READLINE
